### PR TITLE
[chore] - chore: filter non web agent mention

### LINKED ISSUE #5759: Do not count mentions from workflow in assistant usage count
We want to get rid of the mentions that are generated by workflows.

This requires to change the SQL query and the `signalAgentUsage` logic.

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description
 
This PR ensures we only count assistant mentions from actual user (discarding mentions from API calls or workflows)

## Risk

None

## Deploy Plan

Deploy `front`

### CODE REVIEW

File: front/lib/api/assistant/agent_usage.ts [hunk 1]
```diff
@@ -198,6 +198,9 @@ export async function agentMentionsUserCount(
             as: "userMessage",
             attributes: [], // No attributes are necessary here for grouping
             required: true,
+            where: {
+              userContextOrigin: "web",
```
Comment (spolu) [inline front/lib/api/assistant/agent_usage.ts:202]: We likely need an index

If you want slack as well you can search for web or slack

Comment (JulesBelveze) [reply front/lib/api/assistant/agent_usage.ts:202]: Slack includes workflow I guess? In the positive we probably only want to keep web mentions. 

If we take the example of our workspace and include workflows then we end up having the signupRadar assistant as the most popular one (which is not used by anyone)

Comment (JulesBelveze) [reply front/lib/api/assistant/agent_usage.ts:202]: Slack includes workflow I guess? In the positive we probably only want to keep web mentions. 

If we take the example of our workspace and include workflows then we end up having the signupRadar assistant as the most popular one (which is not used by anyone)

**Review by philipperolet — APPROVED** [review]
:+1:

**Pull request merged**